### DISCUSSION TITLE
Error decrypting message : Session key decryption failed

### DISCUSSION THREAD

**Discussion by Abhinav-vats** [openpgpjs/openpgpjs#1499] [Q&A]
@larabr I am using a private key whose public key is shared to the client to decrypt a message which is an API response from client. I can decrypt same response from other pgp tool using private key and passphrase but not on my node js code which uses open pgp js library it is returning following error message:

**Error decrypting message : Session key decryption failed**

FYI: the encryted message has BCPG version 1.6.x, if something related to that.

**Comment by larabr** [comment]
Can you share the encrypted message and public keys that were used?
It might be related to this: https://github.com/openpgpjs/openpgpjs/discussions/1352
